src/app/containers/PodcastPromo/index.test.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
+
+import { ServiceContextProvider } from '#contexts/ServiceContext';
+import PodcastPromo from '.';
+
+/* eslint-disable react/prop-types */
```

Comment:
We could probably test some of the AC from these two issues here: https://github.com/bbc/simorgh/issues/8350 https://github.com/bbc/psammead/issues/3987

some of these AC probably belong in the STY page tests, rather than this component

Reply:
Most of the AC has been tested in the psammead component or the other components that we use eg: `ImageWithPlaceholder`

 I've added AC for positioning to the STY page tests